pkg/driver/external/server/server.go

```diff
@@ -0,0 +1,88 @@
+// SPDX-FileCopyrightText: Copyright The Lima Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package server
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"syscall"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/keepalive"
+
+	"github.com/lima-vm/lima/pkg/driver"
+	pb "github.com/lima-vm/lima/pkg/driver/external"
+)
+
+type DriverServer struct {
+	pb.UnimplementedDriverServer
+	driver driver.Driver
+	logger *logrus.Logger
+}
+
+func Serve(driver driver.Driver) {
+	logger := logrus.New()
+	logger.SetLevel(logrus.DebugLevel)
+
+	socketPath := filepath.Join(os.TempDir(), fmt.Sprintf("lima-driver-%s-%d.sock", driver.GetInfo().DriverName, os.Getpid()))
```

Comment:
Should the socket be created somewhere under `LIMA_HOME`? Maybe `$LIMA_HOME/_run`?

Reply:
I forgot that `/tmp` has the sticky bit set, so I guess this is fine.